nengo/spa/vocab.py

```diff
@@ -379,6 +400,33 @@ def prob_cleanup(self, similarity, vocab_size, steps=10000):
         pcorrect = (1 - perror1) ** vocab_size
         return pcorrect
 
+    def extend(self, keys, unitary=False):
+        """Extends the vocabulary with additional keys.
+
+        Creates and adds the semantic pointers listed in keys to the
+        vocabulary.
+
+        Parameters
+        ----------
+        keys : list of strings
+            List of semantic pointer names to be added to the vocabulary.
+
+        """
+        if is_iterable(unitary):
```

Comment:
That is exactly how it reads. It reflects the behaviour of the ```unitary``` parameter in the constructor where you can either pass it a boolean (in which case, all vectors created will be / not be unitary), or a list of strings (in which case only semantic pointers that are within said list will be unitary).